### PR TITLE
Dev latency

### PR DESCRIPTION
**Pull request by indraniBh**
Added latency stats

### CODE REVIEW

File: result/flow.yaml [hunk 1]
```diff
@@ -113,6 +116,18 @@ components:
...
+          type: integer
+        max_latency:
+          description: >-
+            The maximum latency of valid frames received
+          type: integer
+        avg_latency:
+          description: >-
+            The avarage latency of valid frames received
+          type: number
         loss:
```
Comment (ashutshkumr) [inline result/flow.yaml:131]: We need to specify unit of latency measurements (ns).
@ajbalogh should we include this in attr name `min_latency_ns` ? or just mention the units in description ?

**Comment by ashutshkumr** [comment]
@ajbalogh @winstonliu1111 @winstonliu1111 should we add first and last timestamps as well to flow stats ? usecase is to calculate / validate total duration of each flow.

**Review by ashutshkumr — CHANGES_REQUESTED** [review]
needs changes

File: result/flow.yaml (hunk 1 above)
Comment (ajbalogh) [reply result/flow.yaml:131]: Including units would keep it consistent with bytes_tx, frames_tx etc.

**Review by ajbalogh — CHANGES_REQUESTED** [review]

**Comment by winstonliu1111** [comment]
> @ajbalogh @winstonliu1111 @winstonliu1111 should we add first and last timestamps as well to flow stats ? usecase is to calculate / validate total duration of each flow.

I think we should wait and see how we deal with events and analysis in general as opposed to bake this into a stats path.  That is the next stop we need to model and resolve after we wrap first phase of this srte discussions.

File: result/flow.yaml (hunk 1 above)
Comment (indraniBh) [reply result/flow.yaml:131]: Done

**Review by ashutshkumr — APPROVED** [review]

**Pull request merged**